Xamarin.Forms.Core/Page.cs

```diff
@@ -40,6 +40,8 @@ public class Page : VisualElement, ILayout, IPageController, IElementConfigurati
 		[EditorBrowsable(EditorBrowsableState.Never)]
 		public static readonly BindableProperty IconProperty = IconImageSourceProperty;
 
+		public static readonly BindableProperty ModalBackgroundColorProperty = BindableProperty.Create(nameof(ModalBackgroundColor), typeof(Color), typeof(Page), Color.Default);
```

Comment:
It's uncommon to have a single page used both in modal and non-modal scenarios, so the added property is not required, we could use the "default" one.

in the rare case the user wants to push a modal with a different bg, he knows it at the time of the push, and can change the bg.